app/views/layouts/application.html.erb

```diff
@@ -1,23 +1,21 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>JamaicaContacts</title>
-  <%= csrf_meta_tags %>
-  <%= csp_meta_tag %>
+  <head>
+    <title>JamaicaContacts</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
 
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'https://maps.googleapis.com/maps/api/js?key=AIzaSyAq7YSMMp-zWSRYBhmX59kFXb7yrw_aaOE' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= yield(:head_tags) %>
+  </head>
 
-  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
I fixed the indentation in this file, but the key thing that was changed was the removal of this line. the javascript_pack_tag is only used for webpacker